test/Mesh/testConstraints.jl

```diff
@@ -0,0 +1,13 @@
+
+using jInv.Mesh
+
+h = rand(6)
+msh = getTensorMesh3D(h,h,h)
+
+Ne, Qe, activeEdges = getEdgeConstraints(msh);
+Nf,Qf = getFaceConstraints(msh);
+
+Curl = getCurlMatrix(msh)
+Curl = Qf  * Curl * Ne
+Msig = Ne' * Diagonal(ones(msh.ne)) * Ne
+Mmu  = Nf' * Diagonal(ones(msh.nf))  * Nf
```

Comment:
Yeah, it would be nice to have something more meaningful here but I couldn't really come up with anything.  I'll check with Christoph and see if he has any ideas...